pkg/build/builder/sti.go

```diff
@@ -226,12 +228,11 @@ func (s *S2IBuilder) Build() error {
 
 	// If DockerCfgPath is provided in buildapiv1.Config, then attempt to read the
 	// dockercfg file and get the authentication for pulling the images.
-	t, _ := dockercfg.NewHelper().GetDockerAuth(config.BuilderImage, dockercfg.PullAuthType)
-	config.PullAuthentication = s2iapi.AuthConfig{Username: t.Username, Password: t.Password, Email: t.Email, ServerAddress: t.ServerAddress}
```

Comment:
Delete the comment as well?

(Also noting that the `PullAuthentication` field was not removed, and some code in source-to-image still refers to it. From a very quick look it seems that it can’t be used (it would crash on a `nil` interface submitted by `runtimeBuilderFactory.Builder`), but I may well be not looking hard enough.)

Reply:
I did an detailed examination, and the "as dockerfile" path in source-to-image does not user PullAuthentication.

I believe the comment still has relevance for the subsequent code below.